docs/snapshots.md

```diff
@@ -0,0 +1,54 @@
+# Index Snapshots
+
+This document describes index snapshots, what they are and how they can be created or updated.
+
+**Table of Contents**
+
+- [What Are Index Snapshots](#what-are-index-snapshots)
+- [Creating a Snapshot](#creating-a-snapshot)
+- [Updating a Snapshot](#updating-a-snapshot)
+    1. [New Index](#new-index)
+    2. [New Snapshot](#new-snapshot)
+
+## What Are Index Snapshots
+
+Index snapshots are images of the DPS index at a certain point in time.
+These images can be used to create or restore the content of an index in order to operate on a certain indexed data.
```

Comment:
That sentence doesn't really make sense to me. Something along the lines of `These images can be used to easily transfer DPS index snapshots as a single file or to archive them in a more space-efficient manner` makes more sense to me.